app/controllers/movies_controller.rb

```diff
@@ -48,16 +49,32 @@ def movies_api_key
     ENV['MOVIES_API_KEY']
   end
 
+  def conn
+    Faraday.new(url: "https://api.themoviedb.org")
+  end
+
+  def parse(response)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
   def top_movies_endpoint(page_num)
     sort_by = 'vote_average.desc'
-    conn.get("3/discover/movie?api_key=#{movies_api_key}&#{language('en-US')}&sort_by=#{sort_by}&#{exclude_adult}&page=#{page_num}")
+    conn.get("3/discover/movie?api_key=#{movies_api_key}&#{language('en-US')}&sort_by=#{sort_by}&#{exclude_adult}&page=#{page_num}&vote_count.gte=300")
```

Comment:
Glad we could get this figured out and stop getting weird movies! (rip [nic eats a hamburger](https://www.youtube.com/watch?v=f-1HFJJW8Q0))